app/app/Http/Controllers/LocationReportController.php

```diff
@@ -160,4 +166,56 @@ public function showComments(string $location_id)
         
         return view('pages.location_report.comments', $view_data);
     }
+
+    public function addSuggestion(Request $request)
+    {
+        // user needs to sign in before sending suggestions
+        if (!BaseUser::isSignedIn()) {
+            return redirect()->intended('signin');
+        }
+        $user = BaseUser::getDbUser();
+        // validate data from front end
+        $validation_rules = array(
+            'location-id'           => 'required',
+            'location-name'         => 'required',
+            'phone-number'          => '',
+            'url'                   => '',
```

Comment:
Similar to the phone number validation, check a few things with the URL.  

If the url is empty, that's valid but it it has even 1 character in it, it must be a valid website's URL.   Verify that the url string starts with either http:// or https://.

If you add 'url' to the validation_rules, does it allow empty strings for url?  I hope it does so that can do a simple check for URL-validity.

Here is a screenshot from https://laravel.com/docs/5.6/validation#rule-url:
![image](https://user-images.githubusercontent.com/11838191/42720473-46bc8070-86f5-11e8-9716-25fc4ad785f0.png)